### DISCUSSION TITLE
Fully responsive and support className

### DISCUSSION THREAD

**Discussion by theDanielJLewis** [samhirtarif/react-audio-player#9] [Ideas]
I would love it if the visualizer and audio player would be natively responsive. This would require letting `width` and `height` be optional and adding support for `className` on both `<AudioPlayer>` and `<AudioVisualizer>`.

**Comment by samhirtarif** [comment]
@theDanielJLewis - A great suggestion and this is already on my to-do list 👍🏻

**Reply by samhirtarif** [reply]
I would appreciate if you could share what the audio player looks like for you right now.
And if possible, update its design through devtools and share what you'd roughly like it to look like.
